apps/req.c

```diff
@@ -468,6 +468,10 @@ int req_main(int argc, char **argv)
             break;
         case OPT_MD:
             digest = opt_unknown();
+            if (!EVP_get_digestbyname(digest)) {
```

Comment:
This is wrong. The digest might be also fetchable from a provider.

Reply:
You can use opt_md_silent() instead (and do not forget to EVP_MD_free() the obtained md).

Reply:
No, that won't work. You have to save thye string version and then actually get/fetch.  For example, https://github.com/openssl/openssl/blob/master/apps/cms.c#L710

Reply:
> You can use opt_md_silent() instead (and do not forget to EVP_MD_free() the obtained md).



> Unfortunately this also is not within the limits for CLA: trivial changes. Could you please submit a CLA? https://www.openssl.org/policies/cla.html

Thanks , I try to fix CLA trouble.